StoragePlayer.cs

```diff
@@ -196,14 +196,7 @@ public override bool ShiftClickSlot(Item[] inventory, int context, int slot)
 				return false;
 			int oldType = item.type;
 			int oldStack = item.stack;
-			if (StorageCrafting())
-			{
-				GetCraftingAccess().TryDepositStation(item);
-			}
-			else
-			{
-				GetStorageHeart().TryDeposit(item);
-			}
+            GetStorageHeart().TryDeposit(item);
```

Comment:
Please edit your Pull Requests to have tabs as indents instead of spaces.

Reply:
Done!